docs/anomaly-detection/components/reader.md

```diff
@@ -53,13 +53,13 @@ reader:
 
 ### Per-query parameters
 
-Starting from [v1.13.0](/anomaly-detection/changelog/v1130) there is change of [`queries`](/anomaly-detection/components/reader?highlight=queries#vm-reader) arg format. Now each query alias supports the next (sub)fields:
+Starting from [v1.13.0](/anomaly-detection/changelog#v1130) there is change of [`queries`](/anomaly-detection/components/reader?highlight=queries#vm-reader) arg format. Now each query alias supports the next (sub)fields:
```

Comment:
@fred-navruzov @Cambaza it is recommended using absolute links in documentation - see this guideline https://docs.victoriametrics.com/#documentation